features/file.yml

```
@@ -1,7 +1,12 @@
-draft_date: 2024-10-21
 name: File API
-description: TODO
+description: The `File` API provides information about files and allows JavaScript in a web page to create new files, and access existing files and their contents.
```

We're it not for caniuse and a lot of references to "File API" on Stack Overflow and elsewhere, I'd say this is one of those things that exists spread around the platform (like `ImageData`). So I suggest we try to show that this is an intermediary type:

```suggestion
description: The File API represents and reads from file data and metadata. Typically, you get files through other APIs, such as the `<input type="file">` element or drag-and-drop events. You may also send file data to other APIs, such as creating images or drawing on canvases.
```